app/javascript/shared/components/__tests__/useMediaQuery.test.js

```diff
@@ -0,0 +1,49 @@
+import { renderHook } from '@testing-library/preact-hooks';
+import { cleanup } from '@testing-library/preact';
+import { useMediaQuery } from '@components/useMediaQuery';
+
+describe('useMediaQuery', () => {
+  it('should return false if the media query is not matched', () => {
+    const addListener = jest.fn();
+    const removeListener = jest.fn();
+
+    global.window.matchMedia = jest.fn((query) => {
```

Comment:
Need to mock this as it does not exist in Node as it is related to browser UI.

Reply:
does this automatically get removed from `global.window` after `cleanup` is called? Just double checking we don't need an `afterAll` to make sure it's cleared for subsequent tests

Reply:
I'll double check, but I believe so as it's using jest.fn() and mocks are supposed to test automatically.

Reply:
Confirmed, that it does get removed.